.github/workflows/release.yaml

```diff
@@ -92,45 +111,55 @@ jobs:
       ARTIFACT_DIR: ./release
     runs-on: ubuntu-latest
     steps:
+
+      - name: Checkout code into current directory
+        uses: actions/checkout@v2
+
+      - name: Parse release version and set REL_VERSION
+        run: python ./.github/scripts/get_release_version.py
+
       - name: download artifacts
         uses: actions/download-artifact@master
         with:
           name: bundle_drop
           path: ${{env.ARTIFACT_DIR}}
-      
-      - name: Set Release Version
-        run: |
-          REL_VERSION_FILE="${{ env.ARTIFACT_DIR }}/release_version.txt"
-          REL_VER=`cat ${REL_VERSION_FILE}`
-          echo "REL_VERSION=${REL_VER}" >> $GITHUB_ENV
-          rm -f ${REL_VERSION_FILE}
 
       - name: generate checksum files
         run: cd ${ARTIFACT_DIR} && for i in *; do sha256sum -b $i > "$i.sha256"; done && cd -
 
       - name: lists artifacts
         run: ls -l ${{ env.ARTIFACT_DIR }}
-
-      - name: publish binaries to github
-        if: startswith(github.ref, 'refs/tags/v')
-        run: |
-          echo "installing github-release-cli..."
-          sudo npm install --silent --no-progress -g github-release-cli@1.3.1
-          # Get the list of files
-          RELEASE_ARTIFACT=(${ARTIFACT_DIR}/*)
-          # Parse repository to get owner and repo names
-
-          OWNER_NAME="${GITHUB_REPOSITORY%%/*}"
-          REPO_NAME="${GITHUB_REPOSITORY#*/}"
-
-          export GITHUB_TOKEN=${{ secrets.DAPR_BOT_TOKEN }}
-
-          echo "Uploading Dapr Installer-Bundle to GitHub Release"
-          github-release upload \
-            --owner $OWNER_NAME --repo $REPO_NAME \
-            --tag "v${REL_VERSION}" \
-            --name "Dapr Installer-Bundle v${REL_VERSION}" \
-            --prerelease true \
-            ${RELEASE_ARTIFACT[*]}
+      
+      - name: Upload release using ncipollo/release-action
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "${{ env.ARTIFACT_DIR }}/*"
+          token:  ${{ secrets.DAPR_BOT_TOKEN }}
+          tag: "v${{ env.REL_VERSION }}"
+          name: "Dapr Installer-Bundle v${{ env.REL_VERSION }}"
+          prerelease: ${{ env.PRERELEASE }}
+          allowUpdates: true
```

Comment:
what is this field used for?

Reply:
> what is this field used for?

`allowUpdates` is used to have the support to override the previously released bundle e.g. in case of manually overriding using workflow_dispatch